packages/scandipwa/src/component/FieldNumber/FieldNumber.component.js

```diff
@@ -43,7 +43,7 @@ export class FieldNumber extends PureComponent {
             isDisabled
         } = this.props;
 
-        const numberValue = +value;
+        const numberValue = +attr.value || +value;
```

Comment:
What is the difference between `value` and `attr.value`?
I believe we should have only one prop for value. Having many is confusing - we never know which one is correct.

Reply:
strange but numberField in the PDP uses value, the ones in cartOverlay and cartPage use attr.value. When I tried to pass as a value prop, the numbers stopped changing

Reply:
Please take a look at the field definition.
The `value` prop is followed by the `attr` props.
Thus the `value` will override the `attr.value`.

So please:
1. don't use `attr.value` at all
2. go through all other fields and check that they are using the `value` prop instead of `attr.value` in their definition.
3. go through all the code where fields are rendered and check that none of them is passing value inside the `attr`, but is using the `value` prop instead.

![image](https://user-images.githubusercontent.com/73945186/164342391-dc17df75-f2d5-413c-852e-4a8f1d125d49.png)